src/libs/PolicyUtils.ts

```diff
@@ -269,6 +270,10 @@ function goBackFromInvalidPolicy() {
     Navigation.navigate(ROUTES.SETTINGS_WORKSPACES);
 }
 
+function isPolicyFeatureEnabled(policy: OnyxEntry<Policy> | EmptyObject, featureName: PolicyFeatureName): boolean {
+    return Boolean(policy?.[featureName]);
```

Comment:
This would always evaluate to true for the tax case, since that's an object, wouldn't it?

Reply:
Correct, good catch!
Fixed - c901703.